internal/pkg/registry/ecr.go

```diff
@@ -47,11 +48,12 @@ func IsECR(registry string) (ecr bool, region, account string) {
 }
 
 //
-func newECR(registry, region, account string) ListSource {
+func newECR(registry, region, account string, creds *auth.Credentials) ListSource {
```

Comment:
pass in only the config map, not the full creds; that's all the ecr instance needs to know